src/components/tasks/TaskSearch/tasksearch.module.scss

```diff
@@ -216,9 +209,9 @@
 }
 
 .filter-modal {
-    position: relative;
-    top: 11.5rem;
-    left: -2.8rem;
+    position: absolute;
```

Comment:
This is relative which element

Reply:
It relative to filter button(which contains text filter and chevron). https://github.com/Real-Dev-Squad/website-status/blob/4dc6675e0f83734a631486bceb6db0b1adfcc32c/src/components/tasks/TaskSearch/tasksearch.module.scss#L41